app/templates/components/events/view/overview/event-tickets.hbs

```diff
@@ -33,9 +33,9 @@
       {{#each data.tickets as |ticket|}}
         <tr>
           <td>{{ticket.name}}</td>
```

Comment:
Make changes in `number of orders` `sales amount`, Ticket in the summary section